env.source.sample

```diff
@@ -5,11 +5,12 @@
 ### How its used:
 ###     build.sh: time ${CONTAINER_SUBSYS}  build
 ###     ocm-container.sh: ${CONTAINER_SUBSYS} run -it --rm
+# OCM_CONTAINER_ENV_REQUIRED: change this env to containerization tool
```

Comment:
I was trying to figure out what this env var does and what it's set to, and why it seems to be set multiple times until I realized this is just a flag mentioning that the var below is required?

Maybe we should just change these lines to

```
# REQUIRED: change this env to [whatever]
```

Reply:
yeah, I can change it to `REQUIRED`

Reply:
done :)